remove-api-key updated (#206)
Macrometacorp/docs: docs/sdks/remove-api-key.md

client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                        email='nemo@nautilus.com', password='xxxxx',
                        geofabric='_system')
                        
# Get a list of all the API key ID's names
apiKeyIds = []
for api in client.list_all_api_keys():
    apiKeyIds.append(api["keyid"])

> # Check wether the chosen_api_key exists or not, comparing it to the API key list

Reviewer: ```suggestion
# Check whether the chosen_api_key exists or not, comparing it to the API key list
```

> # Check whether the chosen_api_key exists or not, comparing it to the API key list

Reviewer: Why is chosen_api_key lowercase here and uppercase in line 42?

Author: Hi Diana. PEP8 standard (and JS standard is similar) states that constants should be in upper-case.

We recently changed our code to be as PEP8 compliant as possible.
And certainly, that comment line was bypassed while making the conversion.
Will fix it shortly.

> # Check whether the CHOSEN_API_KEY_ID exists or not, comparing it to the API key list

Reviewer 2: ```suggestion
# Checks if the API key exists
```

for apiElement in apiKeyIds:
    if apiElement == CHOSEN_API_KEY_ID:

>         API_ACTIVE = True

Reviewer 3: @jesusgonzalez12 I guess you forgot to declare CHOSEN_API_KEY_ID and API_ACTIVE on top after import C8Client.
Example:
API_ACTIVE = False
CHOSEN_API_KEY_ID = "YourApiKey"

Reviewer 3: @jesusgonzalez12 The same is missing I guess in "docs/sdks/create-api-key.md" as well you can add the declaration of these two variables there too.

Author: Thanks @koshy-thomas. You're right, thanks for checking. Making changes.

PS: for this I have a single file that adds the key if the ID does not exists while removing it If the key already exists. So translating that to two separate files I had to change a few prints and picky-choose what I needed for each example. As I can see, I have room for improvement on that department.

Changes here are complete. Will now check PR 205.

        break

# Checks the specified API key
if API_ACTIVE:
    # Removes the chosen API key
    try:

>         print("Remove API Key: ", client.remove_api_key(CHOSEN_API_KEY_ID))

Reviewer: ```suggestion
        print("Remove API key: ", client.remove_api_key(CHOSEN_API_KEY_ID))
```

Author: Thanks for the suggestions @diana-macrometa. Updated with suggestions.

    except Exception as err:
        print("Error removing the API key")
        print(err)
else:
    # Displays a message that the key does not exist

>     print("Can't remove a non-existent key")

Reviewer: ```suggestion
    print("Can't remove a non-existent key.")
```

Reviewer: Be sure to finish sentences with periods. Apply throughout.

Author: Thanks Diana.

This writing style is not in the original files.
Good to have it as a way to increase consistency on the docs.
Fixed.

                        email='nemo@nautilus.com', password='xxxxx',
                        geofabric='_system')

> # Check wether the chosen_api_key exists or not, comparing it to the list elements

Reviewer 2: ```suggestion
# Checks if `chosen_api_key` exists
```

What do we mean by "comparing it to the list elements"?

Author: Sorry, I missed adding a few lines of code. Fixed.

A list of all API key IDs a user has is saved in a variable which is used to make a comparison check later.

Author: I have updated the comment to improve semantics.

for apiElement in apiKeyIds:
    if apiElement == CHOSEN_API_KEY_ID:
        API_ACTIVE = True
        break

> # Checks if the API already exists and acts accordingly

Reviewer 2: ```suggestion
# Checks the specified API key
```

If I understand correctly, this step merely checks the API key, and the next one removes it. Correct?

if API_ACTIVE:

>     # Remove the chosen API key

Reviewer 2: ```suggestion
    # Removes the chosen API key
```

    try:
        print("Remove API Key: ", client.remove_api_key(CHOSEN_API_KEY_ID))
    except Exception as err:
        print("Error removing the API key")
        print(err)
else:

>     # Display message that the eky does not exists

Reviewer 2: ```suggestion
    # Displays a message that the key does not exist
```
